src/Accordion/accordion.spec.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Accordion from './accordion';
+import AccordionItem from '../AccordionItem/accordion-item';
+
+jest.mock('../AccordionItem/accordion-item', () => 'div');
```

Comment:
It would be better to use shallow rendering (`shallow` from enzyme with the JSON serialiser works well) – this creates nonsensical snapshots where divs look like they have attributes they shouldn't have, like:

```
+  <div
 +    accordion={true}
 +    disabled={undefined}
 +    expanded={false}
 +    onClick={[Function]}
 +  >
```